### PR TITLE
Fixed unencoded space character in example

### PR DESCRIPTION
**Pull request by kallerosenbaum**

### CODE REVIEW

File: bip-0121.mediawiki [hunk 1]
```diff
@@ -115,7 +115,7 @@ Send PoP for a transaction with label "video 42923" to
 <nowiki>https://www.example.com/pop/352</nowiki>, using nonce
 <tt>0x73 0xd5 0x1a 0xbb 0xd8 0x9c</tt>:
 <pre>
- btcpop:?p=https://www.example.com/pop/352&n=zgWTm8yH&label=video 42923
+ btcpop:?p=https://www.example.com/pop/352&n=zgWTm8yH&label=video%2042923
```
Comment (luke-jr) [inline bip-0121.mediawiki]: The correct way to encode a space is usually '+'

Comment (dabura667) [inline bip-0121.mediawiki]: @luke-jr Are there browsers that don't support %20?

Comment (kallerosenbaum) [inline bip-0121.mediawiki]: Why is '+' a valid encoding for ' ' (space) in bitcoin: URIs? Isn't that a URL encoding thing? I think there is nothing in RFC 3986 about that, nor in BIP21? I think '+' means '+' in RFC 3986.

However '+' is interpreted as space (' ') by most wallets (that I tried), but I do think it's a mistake to do so.

Comment (luke-jr) [inline bip-0121.mediawiki]: AFAIK '+' is the same style of encoding as "%20"

Comment (kallerosenbaum) [inline bip-0121.mediawiki]: I can't find anything in the specifications (BIP21 and RFC 3986) saying that space can be encoded with '+'. Please tell me where I can find that.

**Comment by kallerosenbaum** [comment]
Is there something holding this back?

**Comment by schildbach** [comment]
%20 is fine as well. I'm not sure where the '+' convention comes from, maybe it's just a convention that was never formally standardized.

**Pull request merged**